store/sqlstore/team_store.go

```diff
@@ -6,26 +6,22 @@ package sqlstore
 import (
 	"database/sql"
 	"fmt"
+	"github.com/mattermost/mattermost-server/einterfaces"
 	"net/http"
 	"strings"
 
 	sq "github.com/Masterminds/squirrel"
 	"github.com/mattermost/gorp"
-	"github.com/mattermost/mattermost-server/einterfaces"
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/store"
-	"github.com/mattermost/mattermost-server/utils"
 )
 
 const (
-	TEAM_MEMBER_EXISTS_ERROR         = "store.sql_team.save_member.exists.app_error"
-	ALL_TEAM_IDS_FOR_USER_CACHE_SIZE = model.SESSION_CACHE_SIZE
-	ALL_TEAM_IDS_FOR_USER_CACHE_SEC  = 1800 // 30 mins
+	TEAM_MEMBER_EXISTS_ERROR = "store.sql_team.save_member.exists.app_error"
 )
 
 type SqlTeamStore struct {
 	SqlStore
-	metrics einterfaces.MetricsInterface
```

Comment:
If you going to remove the metrics here, probably you should remove it from the `NewSqlTeamStore` method

Reply:
I did that before but @streamer45 made some suggestions about keeping the signature as before. 

https://github.com/mattermost/mattermost-server/pull/13075#discussion_r347896363

Reply:
So I'll leave it as the first time. It doesn't make sense to me to have non used parameters, specially if we are not implementing any interface